dbtestapp/dbtestapp.go

```diff
@@ -61,7 +74,21 @@ func main() {
 		log.Println("Error getting student: " + err.Error())
 	}
 
-	createDocumentWithTimeout()
+	log.Println("starting timeout document")
+	database := MongoDBLibrary.Client.Database("sdcore")
+	timeoutColl := database.Collection("timeout")
+	//create stream to monitor actions on the collection
+	timeoutStream, err := timeoutColl.Watch(context.TODO(), mongo.Pipeline{})
```

Comment:
Can multiple PODs watch for same collection?

Reply:
Yes they can. tested with multiple dbtestapp pods. But index create might fail for one of the pods cause both will try to create timeout index on same collection with same timeout value. But even if it fails, the one that succeeded would have created the index. so it should be ok.